components/gitpod-db/.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Is there any way this could live in the root of our project?

Reply:
unfortunately not

Reply:
A leeway restriction: For that, all containing resource must be under the root (defined by where the BUILD.yaml resides).

Reply:
Sad. Seems like we're fairly constraint by leeway